templates/donation/informations.html.twig

```diff
@@ -25,74 +27,81 @@
+            <p class="text--left text--smallest">La saisie de votre adresse fiscale exacte est obligatoire pour pouvoir faire un don.</p>
+            {{ form_errors(form.address) }}
 
-        <div class="form__row form__row--stretch">
-            <div class="form--mid">
-                {{ form_errors(form.postalCode) }}
-                {{ form_widget(form.postalCode, { 'attr': { 'class': 'form--full', 'placeholder': 'Code postal' } }) }}
-            </div>
-            <div class="form--mid">
-                {{ form_errors(form.cityName) }}
-                {{ form_widget(form.cityName, { 'attr': { 'class': 'form--full', 'placeholder': 'Ville' } }) }}
+            <div class="form__row address-autocomplete"></div>
+
+            <div class="address-block">
+                <div class="form__row">
+                    {{ form_errors(form.address) }}
+                    {{ form_widget(form.address, { attr: { class: 'form--full', placeholder: 'Adresse postale' } }) }}
+                </div>
+
+                <div class="form__row">
+                    {{ form_errors(form.country) }}
+                    {{ form_widget(form.country, { attr: { class: 'form--full' } }) }}
+                </div>
+
+                <div class="form__row form__row--stretch">
+                    <div class="form--mid">
+                        {{ form_errors(form.postalCode) }}
+                        {{ form_widget(form.postalCode, { attr: { class: 'form--full', placeholder: 'Code postal' } }) }}
+                    </div>
+                    <div class="form--mid">
+                        {{ form_errors(form.cityName) }}
+                        {{ form_widget(form.cityName, { attr: { class: 'form--full', placeholder: 'Ville' } }) }}
+                    </div>
+                </div>
             </div>
         </div>
 
-        <div class="form__row form__row--stretch">
-            {{ form_errors(form.phone) }}
-            {{ form_widget(form.phone.country, { 'attr': { 'class': 'donation__form__phone, form--mid'} }) }}
-            {{ form_widget(form.phone.number, { 'attr': { 'class': 'form--mid', 'placeholder': 'Numéro de téléphone' } }) }}
-        </div>
-
         <div class="form__row form__checkbox form__checkbox--large">
-            <input type="checkbox" required="required" id="donation_check" />
-            <label for="donation_check" id="donation_check_label">
-                Je certifie sur l'honneur être une personne physique et que le règlement de mon don
-                provient de mon compte bancaire personnel ou de celui de mon conjoint, concubin,
-                ascendant ou descendant.
-            </label>
+            {{ form_row(form.isPhysicalPerson, { label: "Je certifie sur l'honneur être une personne physique et que le règlement de mon don provient de mon compte bancaire personnel ou de celui de mon conjoint, concubin, ascendant ou descendant.", label_attr: {id: 'donation_check_label'} }) }}
```

Comment:
Not a big thing but missing white spaces here : `{ id: 'donation_check_label' }`